src/storage/queries.py

```
@@ -1849,3 +1849,410 @@ def response(self, response: Any) -> QueryResult:
+            "summary_id": p.get("summary_id", ""),
+            "content": p.get("content", ""),
+            "level": p.get("level", ""),
+            "source_id": p.get("source_id", ""),
+            "document_id": p.get("document_id", ""),
+            "parent_summary_id": p.get("parent_summary_id", ""),
+            "key_points": p.get("key_points", "[]"),
+            "word_count": p.get("word_count", 0),
+            "created_at": p.get("created_at", int(time.time())),
+        }]
+
+    def response(self, response: Any) -> QueryResult:
```

Ruff (0.14.10)

Unused method argument: `response`

(ARG002)